README.md

```diff
@@ -10,155 +10,71 @@ Changes for 3.x should be made directly to the latest release branch they're
 
 
-
 # OpenShift Ansible
 
 This repository contains [Ansible](https://www.ansible.com/) roles and
 playbooks to install, upgrade, and manage
-[OpenShift](https://www.openshift.com/) clusters.
-
-**Note**: the Ansible playbooks in this repository require an RPM
-package that provides `docker`. Currently, the RPMs from
-[dockerproject.org](https://dockerproject.org/) do not provide this
-requirement, though they may in the future. This limitation is being
-tracked by
-[#2720](https://github.com/openshift/openshift-ansible/issues/2720).
-
-## Getting the correct version
-When choosing an openshift release, ensure that the necessary origin packages
-are available in your distribution's repository.  By default, openshift-ansible
-will not configure extra repositories for testing or staging packages for
-end users.
-
-We recommend using a release branch. We maintain stable branches
-corresponding to upstream Origin releases, e.g.: we guarantee an
-openshift-ansible 3.2 release will fully support an origin
-[1.2 release](https://github.com/openshift/openshift-ansible/tree/release-1.2).
-
-The most recent branch will often receive minor feature backports and
-fixes. Older branches will receive only critical fixes.
-
-In addition to the release branches, the master branch
-[master branch](https://github.com/openshift/openshift-ansible/tree/master)
-tracks our current work **in development** and should be compatible
-with the
-[Origin master branch](https://github.com/openshift/origin/tree/master)
-(code in development).
-
-
-
-**Getting the right openshift-ansible release**
-
-Follow this release pattern and you can't go wrong:
-
-| Origin/OCP    | OpenShift-Ansible version | openshift-ansible branch |
-| ------------- | ----------------- |----------------------------------|
-| 1.3 / 3.3          | 3.3               | release-1.3 |
-| 1.4 / 3.4          | 3.4               | release-1.4 |
-| 1.5 / 3.5          | 3.5               | release-1.5 |
-| 3.*X*         | 3.*X*             | release-3.x |
-
-If you're running from the openshift-ansible **master branch** we can
-only guarantee compatibility with the newest origin releases **in
-development**. Use a branch corresponding to your origin version if
-you are not running a stable release.
+[OpenShift](https://www.openshift.com/) clusters on RHEL hosts.
 
+## Getting Started
 
-## Setup
+### Initial Setup 
 
-Install base dependencies:
+#### Requirements
 
-Requirements:
-
-- Ansible >= 2.6.5, Ansible 2.7 is not yet supported and known to fail
-- Jinja >= 2.7
-- pyOpenSSL
-- python-lxml
-
-----
-
-Fedora:
+[OpenShift Installer](https://github.com/openshift/installer)
 
+[Ansible 2.7](https://docs.ansible.com/ansible/2.5/installation_guide/intro_installation.html) 
 ```
-dnf install -y ansible pyOpenSSL python-cryptography python-lxml
+pip install -U apache-libcloud~=2.2.1
+pip install google-auth
+yum install -y google-cloud-sdk python2-crypto
 ```
 
-Additional requirements:
+##### GCP Credentials
```

Comment:
There shouldn't be any references to GCP bits in the top level readme.

Top level readme should be for end-user deployment steps, not developer steps.  As such, we don't intend for any of the GCP bits to be consumable by end users, only developers.

Reply:
Agreed. Should it live in test/gcp?

Reply:
I think that's a fine place for it.